css/responsive-layout.css

```diff
@@ -0,0 +1,262 @@
+:root{
+    --gap: 15px;    /* --gap yra puse tarpo, susilietus dviem elementam gaunasi pilnas tarpas */
+
+    --col-1: calc((100% - 12 * 2 * var(--gap)) / 12);
```

Comment:
neturetu but 100%-11, nes 11 tu gapu yra?

Reply:
Cia yra matuojamas vieno stulpelio dydis, bendrai stulepliu yra 12, kiekvienas turi po 2 tarpus, is kaires ir is desines.  Apie ka galvoji yra kiek tarpu gride yra tarp stulepiu :)